src/covid_updater/scraping/countries/germany.py

```diff
@@ -15,16 +15,16 @@ def __init__(self):
             },
             column_renaming={
                 "state": "region",
-                "initialDosesCumulative": "people_vaccinated",
-                "finalDosesCumulative": "people_fully_vaccinated",
+                "atLeastPartiallyVaccinatedCumulative": "people_vaccinated",
+                "fullyVaccinatedCumulative": "people_fully_vaccinated",
             },
         )
 
     def load_data(self):
         # Load
         df = pd.read_csv(
             self.data_url,
-            usecols=["date", "state", "initialDosesCumulative", "finalDosesCumulative"],
+            usecols=["date", "state", "atLeastPartiallyVaccinatedCumulative", "fullyVaccinatedCumulative"],
```

Comment:
Include `onlyPartiallyVaccinatedCumulative`, later required to estimate later `total_vaccinations`